lib/resources/example_json/r4_examples_service_request.rb

```diff
@@ -0,0 +1,157 @@
+        }],
+        "text": "Resuscitation Status"
+      },
+      "orderDetail": [
+        {
+          "text": "03/24/20 19:05:00, Full Resuscitation"
+        }
+      ],
+      "quantityQuantity": {
+        "value": 3.0
+      },
+      "subject": {
+        "reference": "Patient/1316020",
+        "display": "Lagunov, Sameera"
+      },
+      "encounter": {
+        "reference": "Encounter/12157884"
+      },
+      "occurrenceDateTime": "2020-03-25T00:05:00.000Z",
+      "asNeededCodeableConcept": {
+        "coding": [
+          {
+            "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/4005",
+            "code": "664912",
+            "display": "inflammation",
+            "userSelected": true
+          },
+          {
+            "system": "http://snomed.info/sct",
+            "code": "1415005",
+            "display": "Inflammation of lymphatics"
+          }
+        ],
+        "text": "inflammation"
+      },
+      "authoredOn": "2020-03-25T00:05:30.000Z",
+      "requester": {
+        "reference": "Practitioner/1626008",
+        "display": "McCready, Tim"
+      },
+      "performer": [
+        {
+          "reference": "Practitioner/664912",
+          "display": "Dr. Spoc"
+        },
+        {
+          "reference": "Practitioner/1744062",
+          "display": "Dr. Dre"
+        }
+      ],
+      "locationReference": [
+        {
+          "reference": "Location/309348",
+          "display": "Mayo Clinic"
+        }
+      ],
+      "reasonCode": [
+        {
+          "text": "Fracture"
+        }
+      ],
+      "bodySite": [
+        {
+          "coding": [
+            {
+              "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/1028",
+              "code": "309344",
+              "display": "Arm L",
+              "userSelected": true
+            },
+            {
+              "system": "http://snomed.info/sct",
+              "code": "362728000",
+              "display": "entire right upper extremity (body structure)"
+            }
+        ],
+        "text": "Arm L"
+        }
+      ],
+      "note": [
+        {
+          "authorReference": {
+            "reference": "Practitioner/1",
+            "display": "Some, Name"
+          },
+          "time": "2014-10-10T15:25:45.000Z",
+          "text": "A comment"
+        }
+      ]
+    }
+
+    R4_SERVICE_REQUEST_BUNDLE ||={
+      "resourceType": "Bundle",
+      "id": "b630105a-2ec0-4877-8f8c-3b26b53cacd9",
+      "type": "searchset",
+      "link": [
+        {
+          "relation": "self",
+          "url":  "https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/ServiceRequest?patient=1316024"
+        }
+      ],
+      "entry": [
+        {
+          "fullUrl": "https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/ServiceRequest/41636441",
+          "resource": R4_SERVICE_REQUEST
+        }
+      ]
```

Comment:
I think this will endup in a weird entry. Can we update it to look something like
```
"entry": [
          R4_SERVICE_REQUEST
      ]
```

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/389/commits/18a592885426eefd181f05aa020842939e654c6a